portfolio/src/main/webapp/style.css

```diff
@@ -15,3 +15,23 @@ img{
     width: 40%;
    
 }
+
```

Comment:
For these selectors, could make the style more consistent so that after each property name there's a space before '{'? I know this is really tedious but when sharing a codebase keeping the styling consistent can help with readability for others and you as well in the future :)